### DISCUSSION TITLE
可以增加每一个笔记有ip地址功能吗🙂

### DISCUSSION THREAD

**Discussion by 707806032** [xisuo67/XHS-Spider#150] [Ideas]
可以添加到文件命名格式那里

**Comment by xisuo67** [comment]
最近996，没有时间维护项目



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年7月11日(周四) 晚上7:37
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [xisuo67/XHS-Spider] 可以增加每一个笔记有ip地址功能吗🙂 (Discussion #150)




 
可以添加到文件命名格式那里

**Reply by 707806032** [reply]
嗯嗯，没关系的，辛苦你啦
